dev/source/docs/building-setup-windows.rst

```diff
@@ -4,7 +4,7 @@
 Setup the Build Environment on Windows
 ======================================
 
-There are three options for building on windows.  We recommended the first option which uses Cygwin.
+There are two options for building on windows.  We recommended the first option which uses Cygwin.
```

Comment:
Perhaps "several" rather than 2. For example, it is possible to set up with the Windows Bash tools too.

Reply:
Fixed